tests/test_no_inference.py

```diff
@@ -0,0 +1,105 @@
+"""
+Set of tests to demonstrate use of Brick *without* the use of a reasoner
+"""
+
+import rdflib
+from rdflib import RDF, RDFS, OWL, Namespace
+
+BRICK_VERSION = '1.1.0'
+
+BRICK = Namespace("https://brickschema.org/schema/{0}/Brick#".format(BRICK_VERSION))
+TAG = Namespace("https://brickschema.org/schema/{0}/BrickTag#".format(BRICK_VERSION))
+BLDG = Namespace("https://brickschema.org/schema/{0}/ExampleBuilding#".format(BRICK_VERSION))
+SKOS = Namespace("http://www.w3.org/2004/02/skos/core#")
+DCTERMS = Namespace("http://purl.org/dc/terms#")
+SDO = Namespace("http://schema.org#")
+A = RDF.type
+
+G = rdflib.Graph()
```

Comment:
Can we standardize the variable name between ``G`` and ``g``? I prefer ``g`` but don't mind using ``G`` too but let's use just one.

Reply:
Changed everything to `g`!